platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/config/ReconnectConfig.java

```diff
@@ -57,10 +57,10 @@ public record ReconnectConfig(
         @ConfigProperty(defaultValue = "false") boolean active,
         @ConfigProperty(defaultValue = "-1") int reconnectWindowSeconds,
         @ConfigProperty(defaultValue = "0.50") double fallenBehindThreshold,
-        @ConfigProperty(defaultValue = "100000") int asyncStreamTimeoutMilliseconds,
-        @ConfigProperty(defaultValue = "100") int asyncOutputStreamFlushMilliseconds,
+        @ConfigProperty(defaultValue = "100000ms") Duration asyncStreamTimeout,
+        @ConfigProperty(defaultValue = "100ms") Duration asyncOutputStreamFlush,
```

Comment:
This must be communicated since a config file that includes a long won't work anymore. Adding @cody-littley

Reply:
I've changed the parser to assume ms if no unit given and added ConfigMappings

Reply:
I think this is an acceptable strategy. The majority of our times are specified in milliseconds (although they are not all consistent).